corehq/apps/sms/api.py

```diff
@@ -202,6 +202,18 @@ def send_sms_to_verified_number(verified_number, text, metadata=None,
     )
     add_msg_tags(msg, metadata)
 
+    msg.custom_metadata = {}
+    for event in events:
+        multimedia_fields = ('caption_image', 'caption_audio', 'caption_video')
+        for field in multimedia_fields:
+            try:
+                value = getattr(event, field)
+                if value is not None:
+                    msg.custom_metadata[field] = value
+            except AttributeError:
+                pass
```

Comment:
I think this can be simplified
```suggestion
            value = getattr(event, field, None)
            if value is not None:
                msg.custom_metadata[field] = value
```